modules/opensearch/includes/opensearch.search.inc

```diff
@@ -110,6 +110,8 @@ function opensearch_search_collection_load($id) {
  * @return \Ting\Search\TingSearchResultInterface
  *   The search result.
  *
+ * @throws \TingClientException
+ *   If an search error occurs.
```

Comment:
I do not think such an exception should be thrown from the provider - or at least declared.

The caller should not be aware of what provider is being used and thus not be aware of providerspecific exceptions. 

If we want to handle this properly then we should wrap the location where this exception can be thrown in a `try/catch` and then wrap the `TingClientException` in a `SearchProviderException`.

Reply:
Okay, just filled it in as the IDE was complaining that the function could throw it.

Reply:
So this may be outside the scope of this PR.

Reply:
Then I think it should be removed.

Reply:
But removing the comment don't change the fact that this function can throw the exception. So in-complete documentation should not hide that fact.